doc/user/USAGE.md

```diff
@@ -47,3 +47,16 @@ The **full scene** formats (.gltf/.glb, .3ds, .wrl, .obj, .fbx, .dae, .off) cont
 but also some scene information like *lights*, *cameras*, *actors* in the scene, as well as *texture* properties.
 By default, all this information will be loaded from the file and displayed. Use the `--geometry-only` [options](OPTIONS.md)
 to modify this behavior. For file formats that do not support it, **a default scene** is created.
+
+## Plugins
+
+If you installed F3D using a package manager, it's possible that the packager chose to bundle the plugins in different packages to reduce the reduce the number of dependencies of the main package.
+In this case, you will have to specify which plugin you want to load in order to extend the number of supported file format. You can either use the `--load-plugin` option or add a line in the [configuration file](CONFIGURATION_FILE.md).
+Here is the list of plugins provided officially by F3D:
```

Comment:
we could add them to config.json anyway ? not sure this is a good idea.

Reply:
in any case, here is a rewrite:

```
If you installed F3D using a package manager, it's possible that the packager chose to bundle the plugins in different packages or to list plugin dependencies as optional dependencies to reduce the reduce the number of dependencies of the main package.
In this case, in order to open file that require a plugin, you will have to make sure all needed dependencies are installed and specify which plugin you want to load in order to be able to open this file. You can either use the `--load-plugin` option or add a line in the [configuration file](CONFIGURATION_FILE.md), if not already.
Here is the list of plugins provided officially by F3D:
```

Reply:
> we could add them to config.json anyway ? not sure this is a good idea.

It means we have to `configure_file` the config.json file. It's overkill in my opinion.

Reply:
Why is that ? `load-plugin=assimp` should be enough ?

Reply:
Ok, I understand what you mean. The problem is that packagers will need to distribute or patch the config they install.

Reply:
Yes, it looks too complicated to handle this.

Reply:
We will see what packagers thinkgs about it